docs/pages/setup/deployments/aws-terraform.mdx

```diff
@@ -714,7 +714,7 @@ To add new nodes/EC2 servers that you can "SSH into" you'll need to:
 You can use this command to get the CA pin hash for your Teleport cluster:
 
 ```code
-$ aws ssm get-parameter --name "/teleport/${TF_VAR_cluster_name}/ca-pin-hash" --query "Parameter.Value" --output text
+$ aws ssm get-parameter --region ${TF_VAR_REGION} --name "/teleport/${TF_VAR_cluster_name}/ca-pin-hash" --query "Parameter.Value" --output text
```

Comment:
Why is `TF_VAR_REGION` all caps but `TF_VAR_cluster_name` mixed?

Reply:
Good catch, fixed